packages/env/lib/build-docker-compose-config.js

```diff
@@ -54,6 +54,11 @@ function getMounts(
 			? `user-home:/home/${ hostUsername }`
 			: `tests-user-home:/home/${ hostUsername }`;
 
+	const userUploadsMount =
```

Comment:
I'm sorry, I don't really know that much about docker so I may be misunderstanding. This function creates a `new Set()` which, by definition, doesn't contain duplicates, so I don't think there could already be an entry for it?

Reply:
Let's take, for example, a `wp-env.json` file containing this:

```json
{
  "mappings": {
    "/var/www/html/wp-content/uploads": "./env/uploads"
  }
}
```

This would translate to a unique entry in the Set, but, be a duplicate mapping for `/var/www/html/wp-content/upload`.